notebooks/2.modeling/nbconverted/2.modeling.py

```diff
@@ -18,53 +25,120 @@
 # import local modules
 sys.path.append("../../")
 from src.utils import (
-    check_feature_order,
```

Comment:
why was this dropped? Wasn't this one of the reviewer comments to confirm this? Does dropping this mean we need to update our response?

Reply:
There's no need for an update! I likely must have accidentally removed it during the notebook refactoring. `pycln` automatically removes unused imports, and this one was probably removed for that reason. Great catch, thank you!

Reply:
Fixed now c: